MaxMind.MinFraud/Response/IIPAddress.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace MaxMind.MinFraud.Response
+{
+    /// <summary>
+    /// An interface for IP address risk classes.
+    /// </summary>
+    public interface IIPAddress
+    {
+        /// <summary>
+        /// The risk associated with the IP address. The value ranges from 0.01
+        /// to 99. A higher score indicates a higher risk.
```

Comment:
I think Intellisense does show the description from the interface, but I don't think our Mono documentation tool does. Perhaps we can look into improving that in the future.